CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ _2020-07-15_
 * Fix WorkflowRxSwift podsepcs (#45)
 * Constrain Worker Rendering type to Void (#44)
 * Remove unncessary dependencies (#47)
+* Add `makeOutputSink` convenience on `RenderContext`  
```

Comment:
This isn’t in alpha 4, so we shouldn’t add it here. We’ll add it to the changelog when we cut the next release.